backend/controller/ingress/request_test.go

```diff
@@ -177,7 +177,8 @@ func TestBuildRequestBody(t *testing.T) {
 				Verb:   test.verb,
 			}, r, sch)
 			if test.err != "" {
-				assert.EqualError(t, err, test.err)
+				assert.Contains(t, err.Error(), test.err)
+				assert.IsError(t, err, schema.ErrNotFound)
```

Comment:
`ErrNotFound` won't be valid for all errors, and `assert.EqualError()` should still be used.

Reply:
> ErrNotFound won't be valid for all errors
I misread the test. fixing it.

> and assert.EqualError() should still be used.
So you want to test a match on the error string of a composed error? isn't testing the right kind of error and some key pieces of the message enough? I think the exact matching string is too strict, but happy to do it your way.

Reply:
Yes it should still be used.